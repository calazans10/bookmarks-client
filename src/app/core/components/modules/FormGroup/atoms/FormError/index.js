import React from 'react';
import PropTypes from 'prop-types';

const FormError = ({ children }) => <span className="form-group__error">{children}</span>;

FormError.propTypes = {
  children: PropTypes.node.isRequired,
};

export default FormError;
