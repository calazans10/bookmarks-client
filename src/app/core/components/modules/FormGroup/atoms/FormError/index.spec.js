import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import FormError from './index';

describe('FormError', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<FormError>A message</FormError>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
