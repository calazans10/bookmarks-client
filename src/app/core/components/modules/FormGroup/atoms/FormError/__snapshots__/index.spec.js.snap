// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormError renders without crashing 1`] = `
<span
  className="form-group__error"
>
  A message
</span>
`;
